pyproject.toml

```diff
@@ -46,7 +46,7 @@ invoke = "^1.3"
 isort = "^4.3"
 livereload = "^2.6"
 markdown = "^3.1.1"
-pytest = "^5.0"
+pytest = "5.3.5"
```

Comment:
Because with the latest version of pytest (5.4.1) there is a bug which preaks with pytest_sugar (see: https://github.com/pytest-dev/pytest/issues/6951) So I put the highest version of pytest (that I could find) that made the plug-in work. Or else the plug-in will stop working if people upgrade pytest.

Reply:
Indeed. See also: https://github.com/Teemu/pytest-sugar/issues/187 That said, rather than hard-pinning, it might be better to instead ensure the problematic versions are not installed, via something like:

     pytest = "<5.4,>=5.4.2"

Reply:
Since I wanted to get the tests to pass in the CI-related PR I just submitted (#3), I went ahead and made the above modification. So you should back out your change to this line in your PR's branch and rebase on current master.